src/app/icd/client/DefaultCheckInDelegate.cpp

```diff
@@ -44,5 +43,42 @@ void DefaultCheckInDelegate::OnCheckInComplete(const ICDClientInfo & clientInfo)
 #endif
 }
 
+RefreshKeySender * DefaultCheckInDelegate::OnKeyRefreshNeeded(ICDClientInfo & clientInfo, ICDClientStorage * clientStorage)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    auto refreshKeySender = chip::Platform::New<RefreshKeySender>(this, clientInfo, clientStorage);
```

Comment:
```suggestion
    auto refreshKeySender = Platform::New<RefreshKeySender>(this, clientInfo, clientStorage);
```

This is all inside `namespace chip`, no?

Reply:
Applied.